README.md

```diff
@@ -79,16 +79,29 @@ pipe(['google', 'twitter', 'yahoo', 'facebook', 'github'])
 # similar to `range(1, 11)`,  but this is an iterator.
 # Python's nasty range() is right-exclusive. This is right-inclusive.
 
+1 /to/ 10 /by/ 2
+# similar to `range(1, 11, 2)`, you can also specify step size
+
 10 /to/ 1
 # similar to `range(10, 0, -1)`, you can also use decreasing ranges
 
+10 /to/ 1 /by/ -2
```

Comment:
@terciodemelo Do you mind also to change the doc to positive numbers here and below too?